packages/plugin-vue/src/utils/descriptorCache.ts

```diff
@@ -47,15 +47,16 @@ export function setPrevDescriptor(
 export function getDescriptor(
   filename: string,
   options: ResolvedOptions,
-  createIfNotFound = true
+  createIfNotFound = true,
+  readFile = () => fs.readFileSync(filename, 'utf-8')
```

Comment:
This change could be removed, no? Only the change in index.ts is needed in the PR.